magda-web-client/src/Components/FeedbackForm.js

```diff
@@ -56,6 +60,7 @@ class FeedbackForm extends React.Component {
          * Instead of having button disabled, we will check here to make sure feedback is only sent when no validation error.
          */
         if (this.state.validationErrorMessage) return;
+        this.props.sendFeedbacks();
```

Comment:
This'll work but I think it's more idiomatic redux to do it in the `fetchFeedback` function in `feedbackActions.js` - put `dispatch(sendFeedbacks())` at the start the same way as `dispatch(requestDataset(id));` is at the start of `fetchDatasetFromRegistry` in `recordActions.js`